### PR TITLE
Re-activate and Update Glasgow.gov.uk.xml

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/Glasgow.gov.uk.xml [hunk 1]
```diff
@@ -1,61 +1,85 @@
...
-<ruleset name="Glasgow.gov.uk (partial)" default_off='failed ruleset test'>
-
+<ruleset name="Glasgow.gov.uk">
 	<target host="glasgow.gov.uk" />
-	<target host="data.glasgow.gov.uk" />
-	<target host="onlineservices.glasgow.gov.uk" />
-	<target host="publicaccess.glasgow.gov.uk" />
 	<target host="www.glasgow.gov.uk" />
+	<target host="SPX.glasgow.gov.uk" />
+	<target host="SPX2.glasgow.gov.uk" />
```
Comment (J0WI) [inline src/chrome/content/rules/Glasgow.gov.uk.xml]: URLs are not case sensitive. Seems like our tests do not check this.

File: src/chrome/content/rules/Glasgow.gov.uk.xml [hunk 2]
```diff
@@ -1,61 +1,83 @@
...
-	Insecure cookies are set for these hosts:
-
-		- glasgow.gov.uk
-		- onlineservices.glasgow.gov.uk
-		- www.glasgow.gov.uk
-
+	Non-functional hosts
+		Timeout was reached:
+		- CPMESSEG02S.glasgow.gov.uk
+		- CPMESSEG04S.glasgow.gov.uk
```
Comment (J0WI) [inline src/chrome/content/rules/Glasgow.gov.uk.xml]: Same here

File: src/chrome/content/rules/Glasgow.gov.uk.xml [hunk 3]
```diff
@@ -1,61 +1,83 @@
...
+		- dataservices.open.glasgow.gov.uk
+		- developers.open.glasgow.gov.uk
+		- optic.open.glasgow.gov.uk
+		- services.glasgow.gov.uk
+
+		Incomplete certificate chain error:
+		- CSdev.glasgow.gov.uk
+		- CSprod.glasgow.gov.uk
+		- CSstage.glasgow.gov.uk
+		- CStest.glasgow.gov.uk
```
Comment (J0WI) [inline src/chrome/content/rules/Glasgow.gov.uk.xml]: And here

**Comment by koops76** [comment]
Add `<securecookie host=".+" name=".+" />`.

**Comment by cschanaj** [comment]
@koops76 usually I try to avoid the wildcard `securecookie` tag when the number of subdomain not supporting HTTPS is comparably large. Since this is a governmental site, I have added the `securecookie` this time. thanks!

**Pull request merged**